app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -526,6 +617,21 @@ impl Model {
         Vector2(pos.x, y)
     }
 
+    /// Calculate the view-local position of the selection from the component group-local one.
```

Comment:
Could you expand these docs please? I was reading this sentence few times and I dont really understand what it does.

Reply:
Expanded